stacks/ibm/application/katib-controller/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
For V3 version of Katib, you can easily use this new installer [`installs/standalone`](https://github.com/kubeflow/manifests/blob/master/katib/installs/katib-standalone/kustomization.yaml).
That should follow all V3 version patterns and installs all required Katib components.
I will create PR soon to add Katib to GCP stacks to show how we can do it.

Reply:
Thanks @andreyvelich I had looked at that dir before and I was going to use `installs/standalone-ibm` actually but at that time the crd was not part of it (ie. before your latest commit).

Also may I suggest that we have a README for each application that already has a v3 version?

Reply:
Sure @adrian555, you can see how we added them to generic stacks now: https://github.com/kubeflow/manifests/blob/master/stacks/generic/kustomization.yaml#L23.

I am fine with README under `/katib/` folder. We can say which folders related to old Kubeflow manifest implementation and which to the new V3 version. Also, we can say there what does each `installs` mean: https://github.com/kubeflow/manifests/tree/master/katib/installs.

Should we add this information to `kubeflow/manifests/katib` or it should be part of Kubeflow website @jlewi ?

Reply:
I made the change to use the `katib/installs/generic` as the base.